### PR TITLE
Define dict method to get child or data item

### PR DESCRIPTION
**Pull request by jscotka**

### CODE REVIEW

File: tests/unit/test_items.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+EXAMPLES = PATH + "/../../examples/"
+
+
+class TestGetItems(unittest.TestCase):
+    """ Verify getter of items """
+
+    def setUp(self):
+        self.wget_path = EXAMPLES + "wget"
+        self.wget = Tree(self.wget_path)
+
```
Comment (lukaszachy) [inline tests/unit/test_items.py:20]: Bad formatting: Too many blank lines

File: tests/unit/test_items.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    def test_item_child(self):
+        item = self.wget['/recursion']['/deep']
+        self.assertEqual(item.get("depth"), 1000)
```
Comment (lukaszachy) [inline tests/unit/test_items.py:24]: Could you assert also 'name' ?
`assert self.wget['/recursion']['/deep'].name == '/recursion/deep' `

**Review by lukaszachy — CHANGES_REQUESTED** [review]

**Review by lukaszachy — APPROVED** [review]

**Comment by lukaszachy** [comment]
@psss May I squash and merge this?

File: fmf/base.py [hunk 3]
```diff
@@ -696,3 +696,15 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         _, full_data, source = self._locate_raw_data()
         with open(source, "w", encoding='utf-8') as file:
             file.write(dict_to_yaml(full_data))
+
+    def __getitem__(self, key):
+        """
+        Dictionary method to get child node or data item
+
+        get children have to start with '/'
```
Comment (psss) [inline fmf/base.py:704]:
```suggestion
        To get a child the key has to start with a '/'.
```

**Review by psss — APPROVED** [review]
Looks good, just propose a minor docstring enhancement.

**Pull request merged**